src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+
+const sortSex = (gender) => {
+  if (gender === 'm') {
+    return 'wife';
+  }
+
+  return 'husband';
+};
+
+export const Person = ({ person }) => (
+  <div className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <div
+      className="Person__age"
+      style={{
+        display: person.age <= 0 ? 'none' : undefined,
+      }}
+    >
+      {person.age > 0
+
+        ? (`I am ${person.age}`)
+
+        : undefined}
+    </div>
+    <p className="Person__partner">
+      {person.isMarried === true
+        ? (`${person.partnerName} is my ${sortSex(person.sex)}`)
+        : ('I am a not married')}
+    </p>
```

Comment:
no needs to equal with `true` and `()` symbols are redudnant 
```suggestion
    <p className="Person__partner">
      {isMarried
        ? `${partnerName} is my ${sortSex(sex)}`
        : 'I am a not married'}
    </p>
```